docs/source/reservations/existing-machines.rst

```diff
@@ -316,6 +318,9 @@ Volume mounting can be done directly in the task YAML on a per-task basis, or gl
                        path: /mnt/<mount_tag> # e.g. /mnt/filesystem-d0
                        type: Directory
 
+.. note::
+
+   The ``ssh`` section in ``~/.sky/config.yaml`` applies to every cluster launched on SSH node pools. To mount different paths per cluster, set the ``ssh`` config in the task YAML file as described in the per-task configuration section.
```

Comment:
We should not say `~/.sky/config.yaml` but should link to the advanced config page, as we are recommending people to use the dashboard to add configs.